sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -9,29 +10,30 @@
         <field name="priority">20</field>
         <field name="arch" type="xml">
             <tree
-                decoration-info="invoice_status=='to invoice'"
-                decoration-warning="state=='draft'"
-                decoration-muted="state=='cancel'"
-                decoration-success="state=='done'"
+                decoration-info="invoice_status =='to invoice'"
+                decoration-warning="state =='draft'"
+                decoration-muted="state =='cancel'"
+                decoration-success="order_locked == True"
```

Comment:
I'd simply change this to decoration-success="invoice_status =='invoiced'" and get rid of the `order_locked` stuff

Reply:
Yes, because the locked field of sale.order is a flag that is controlled by the setting of blocking confirmed sales orders, so it is not valid to use it as a flag. Thanks

Reply:
I have added the changes and also reverted the pre-commit changes in another module.